docs/systemtest.rst

```diff
@@ -18,13 +18,17 @@ Note: crashme is `currently not working on Mac OS X <https://bugzilla.mozilla.or
 
 Also see: https://developer.mozilla.org/en/Environment_variables_affecting_crash_reporting
 
-If you already have a crash available and wish to submit it, you can
-use the standalone submitter tool (there is an example JSON and dump
-file for your crash are in the "./testcrash" directory you can use)
-::
-  python socorro/collector/submitter_app.py -u http://crash-reports/submit -s ./testcrash/raw/
+You can use the "curl" command to submit a test crash. If you don't
+have a minidump available, you can download a test dump from the
+Socorro repo:
+.. code-block:: bash
+  curl https://raw.githubusercontent.com/mozilla/socorro/master/testcrash/raw/7d381dc5-51e2-4887-956b-1ae9c2130109.dump > test.dump
```

Comment:
Redirecting STDOUT in this case can have unintended side-effects.  I admit that we're talking weird shell and environment edge cases here, but it's safer to use one of cURL's `-o` or `-O` arguments for this task.